core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -459,4 +495,10 @@ public DDF sort(List<String> columns, List<Boolean> ascending, Boolean inPlace)
       return castType(column, newType);
     }
   }
+
+  private boolean isColumnEligibleToScale(Column column, List<String> scaleRequestedColumnNames) {
+    return (column.isNumeric()
+            && column.getColumnClass() != ColumnClass.FACTOR
+            && ((scaleRequestedColumnNames == null) || scaleRequestedColumnNames.isEmpty() || scaleRequestedColumnNames.contains(column.getName())));
```

Comment:
This has been refactored in DDF, pAnalytics, PyClient, and RClient. @ubolonton @hai-adatao